monitoring/metrics/in-servicepulse.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Performance Metrics in ServicePulse
+summary: Describes how to use ServicePulse to review endpoint performance metrics
+reviewed: 2018-01-12
+---
+
+ServicePulse collects displays performance monitoring data about running endpoints on the Monitoring tab.
+
+![ServicePulse monitoring tab](servicepulse-monitoring-tab.png)
+
+NOTE: ServicePulse will only display the monitoring tab if it is configured to point to a ServiceControl monitoring instance.
+
+The main monitoring tab shows a list of logical endpoints as well as the performance data collected about those endpoints. If a logical endpoint is running more than one physical instance, a badge will show a count of physical instances.
+
+![ServicePulse monitoring tab instance count](servicepulse-instance-count.png)
+
+Clicking on an endpoint in the main monitoring tab will open the endpoint details page.
+
+![ServicePulse monitoring details page](servicepulse-monitoring-details.png)
+
+The endpoint details page shows larger graphs for each of the performance metrics gathered for the logical endpoint. It also contains a number of breakdown views. 
+
+The default breakdown view is by Message Type.
+
+![ServicePulse details breakdown by message type](servicepulse-messagetype-breakdown.png)
+
+This shows a breakdown of performance metrics for each type of message that the endpoint processes.
+
+The second breakdown view is by Instance.
+
+![ServicePulse details breakdown by physical instance](servicepulse-physicalinstance-breakdown.png)
```

Comment:
Also no image?

Reply:
And again. Uppercase .PNG means that GH couldn't find it. I fixed it